core/pom.xml

```diff
@@ -27,16 +27,17 @@
             <artifactId>icedtea-web-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>xml-parser</artifactId>
```

Comment:
since our group-id is `net.adoptopenjdk` for now I would love to add the `icedtea-web-` prefix to modules like this once since this is not "the adopt openJDK xml parser".
Maybe we should with to groupId `net.adoptopenjdk.icedteaweb`...

Reply:
good point, done.